convert/src/label_studio_to_badgerdoc/label_studio_to_badgerdoc_use_case.py

```diff
@@ -0,0 +1,431 @@
+    ) -> None:
+        with tempfile.TemporaryDirectory() as tmp_dirname:
+            tmp_dirname = Path(tmp_dirname)
+
+            s3_output_tokens_path = self.get_output_tokens_path(
+                file_id_in_assets
+            )
+
+            badgerdoc_tokens_path = tmp_dirname / Path("badgerdoc_tokens.json")
+            self.badgerdoc_format.export_tokens(badgerdoc_tokens_path)
+            self.s3_client.upload_file(
+                str(badgerdoc_tokens_path),
+                self.s3_output_bucket,
+                s3_output_tokens_path,
+            )
+
+            badgerdoc_annotations_path = tmp_dirname / Path(
+                "badgerdoc_annotations.json"
+            )
+            self.badgerdoc_format.export_annotations(
+                badgerdoc_annotations_path
+            )
+            s3_output_annotations_path = self.get_output_annotations_path(
+                importjob_id_created, file_id_in_assets
+            )
+            self.s3_client.upload_file(
+                str(badgerdoc_annotations_path),
+                self.s3_output_bucket,
+                s3_output_annotations_path,
+            )
+
+    def request_jobs_to_create_annotation_job(
+        self,
+        file_id_in_assets,
+        owner,
+        validation_type,
+        deadline,
+        extensive_coverage,
+        annotators,
+        validators,
+    ) -> int:
+        categories = self.request_annotations_to_post_categories()
+        post_annotation_job_url = f"{settings.job_service_url}create_job/"
+        post_annotation_job_body = {
+            "name": f"import_label_studio_job_{uuid4()}",
+            "type": "AnnotationJob",
+            "files": [file_id_in_assets],
+            "datasets": [],
+            "is_draft": False,
+            "validation_type": validation_type,
+            "annotators": annotators,
+            "owners": [owner],
+            "categories": categories,
+            "validators": validators,
+        }
+        if deadline:
+            post_annotation_job_body.update(
+                {"deadline": jsonable_encoder(deadline)}
+            )
+        if extensive_coverage is not None:
+            post_annotation_job_body.update(
+                {"extensive_coverage": extensive_coverage}
+            )
+        LOGGER.debug(
+            "Making a request to create an Annotation Job in 'jobs' to url: %s with request body: %s",
+            post_annotation_job_url,
+            post_annotation_job_body,
+        )
+
+        try:
+            request_to_post_annotation_job = requests.post(
+                url=post_annotation_job_url,
+                headers=self.request_headers,
+                json=post_annotation_job_body,
+            )
+            request_to_post_annotation_job.raise_for_status()
+        except requests.exceptions.RequestException as e:
+            LOGGER.exception(
+                "Failed request to 'jobs' to post AnnotationJob - %s", e
```

Comment:
No need to log exception twice
```suggestion
                "Failed request to 'jobs' to post AnnotationJob"
```

Reply:
done